app/queue/egress.py

```diff
@@ -26,11 +26,11 @@ def __init__(self, config):
         self._kafka_producer = KafkaProducer(bootstrap_servers=config.bootstrap_servers)
         self._topic = config.host_egress_topic
 
-    def write_event(self, event):
-        logger.debug(f"Topic: {self._topic} => {event}")
+    def write_event(self, event, host_id):
+        logger.debug(f"Topic: {self._topic}, Key: {host_id} => {event}")
```

Comment:
With _logger_, please use it’s own string interpolation instead of Python’s native formatting. In that case, objects are not stringified if the logging does not take place. Also, the message format is a bit strange – the key has a prefix, but the event itself doesn’t.

```suggestion
        logger.debug(f"Topic: %s, key: %s, event: %s", self._topic, key, event)
```